updater/osv/severity.go

```diff
@@ -0,0 +1,49 @@
+// Package osv is an updater for OSV-formatted advisories.
+package osv
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/quay/claircore"
+)
+
+// check if severity (lower case) exists in the Database object if so, use it
+func extractSeverityFromDatabase(defaultSev claircore.Severity, a advisory) (sev claircore.Severity) {
```

Comment:
I'd say don't pass in the default severity, either let the `Insert` method handle it or return `claircore.Unknown` as the default case in `severityFromString()` like the other `NormalizeSeverity` functions do so it's easier to follow. Also, I'd probably pass in the advisory.Database as the function is called `extractSeverityFromDatabase()`.

AFAICT the named return param is not used.